src/cmd/seal.sh

```diff
@@ -21,18 +21,19 @@ cmd_seal() {
 
     # get recipients
     get_gpg_key
-    local recipients="--recipient $GPG_KEY"
+    local recipients="$GPG_KEY"
     while [[ -n "$1" ]]; do
-        recipients="$recipients --recipient $1"
+        recipients="$recipients $1"
         shift
     done
```

Comment:
In bash this `while` can be done like this:
`local recipients=("$GPG_KEY" "$@")`
This is an array of all the arguments: `"$@"` (you can look it up on internet what it means). We prepend to it $GPG_KEY. It's true that arrays in bash are a bit awkward and cryptic. I always have to look up their usage on internet and then do proper testing, before using them, otherwise it is almost certain that I will do it wrong.

Also you have to be careful and use this array with double quotes, like this:
`call_gpg seal.py "$file" "$recipients"`
In bash it makes a difference when you use double quotes, single quotes, or no quotes at all. Especially for the arrays. The right way to pass arrays as arguments is with double quotes (you can look it up on the book or on the internet why).